### ISSUE TITLE
Debian 9.4 - Makefile:179: recipe for target 'audio/midi_alsa.o' failed

### ISSUE THREAD

**Issue by gbajson** [OpenDUNE/OpenDUNE#318]
Hi,
I have a problem with compilation on Debian 9.4, could you help, please?

```
gcc version 6.3.0 20170516 (Debian 6.3.0-18+deb9u1) 

gbajson@misio:/var/tmp/OpenDUNE$ ./findversion.sh 
ge4e1470f		0	ge4e1470f

```

```
gbajson@misio:/var/tmp/OpenDUNE$ make
make[1]: Entering directory '/var/tmp/OpenDUNE/objs/release'
[SRC] Compiling and linking depend
[SRC] DEP CHECK (all files)
[SRC] Compiling animation.c
[SRC] Compiling audio/driver.c
[SRC] Compiling audio/dsp_pulse.c
[SRC] Compiling audio/midi_alsa.c
In file included from /usr/include/alsa/asoundlib.h:54:0,
                 from /var/tmp/OpenDUNE/src/audio/midi_alsa.c:8:
/usr/include/alsa/pcm.h:1010:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 static inline void snd_pcm_pack_audio_tstamp_config(unsigned int *data,
               ^~~~
/usr/include/alsa/pcm.h:1018:15: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘void’
 static inline void snd_pcm_unpack_audio_tstamp_report(unsigned int data, unsigned int accuracy,
               ^~~~
Makefile:179: recipe for target 'audio/midi_alsa.o' failed
make[1]: *** [audio/midi_alsa.o] Error 1
make[1]: Leaving directory '/var/tmp/OpenDUNE/objs/release'
Makefile:43: recipe for target 'all' failed
make: *** [all] Error 1

```

**Comment by miniupnp** [comment]
I think "-ansi -pedantic" CFLAGS should be disabled.

**Comment by gbajson** [comment]
Thanks, I commented it in config.lib and it solved the problem.

**Comment by miniupnp** [comment]
I think it was fixed by 508e54c0bd67549a1268db2079442ba2f358fd04

I just tested on a debian 9.12 and it builds fine
